nav2_theta_star_planner/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>nav2_theta_star_planner</name>
+  <version>0.4.0</version>
+  <description>Theta* Global Planning Plugin</description>
+  <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
+  <maintainer email="anshumaan567@gmail.com">Anshumaan Singh</maintainer>
+  <license>Apache-2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <depend>rclcpp</depend>
```

Comment:
Please do a quick second check that your dependency are called out here (or its going to cause me a headache later in releasing). Any `find_package` in CMakeLists (including anything you do for testing)

Reply:
(like gtest appears to be missing, at minimum)